src/data/actions/assignmentRevoke.js

```diff
@@ -0,0 +1,47 @@
+import {
+  ASSIGNMENT_REVOKE_REQUEST,
```

Comment:
nit: Rename these constants to `CODE_REVOKE_XX` and rename file to `codeRevoke.js`